src/components/Formatters/useNumberFormatter.tsx

```diff
@@ -1,4 +1,5 @@
 export const useNumberFormatter = (options?: Intl.NumberFormatOptions) => {
   const formatter = new Intl.NumberFormat(undefined, options);
-  return (value: number) => formatter.format(value);
+  return (value: number | undefined | null) =>
+    value ? formatter.format(value) : '';
```

Comment:
Careful with zeros here. Probably need `value != null ?`

Reply:
I forgot zero is falsy in js. I added in an explicit check for null and undefined